.github/workflows/type-tests.yml

```diff
@@ -2,34 +2,29 @@ name: Type Tests
 
```

Comment:
why is this file being modified for a change to linting?

Reply:
OK. Let me revert most unnecessary changes. It's a on-the-fly changes to keep the same CI workflow structure as linting.yml.